docs/concepts/endpoint.md

```diff
@@ -0,0 +1,38 @@
+---
+id: endpoint
+title: Endpoint
+---
+
+Endpoints in ZIO HTTP represent individual API operations or routes that the server can handle. They encapsulate the structure and behavior of the API endpoints in a type-safe manner, making them fundamental building blocks for defining HTTP APIs.
```

Comment:
It is good to focus on the goal of `Endpoint` API‌ which tries to separate the definition of API from its implementation.